soccer/radio/SimRadio.cpp

```diff
@@ -1,12 +1,16 @@
 #include "SimRadio.hpp"
 
+#include <iostream>
 #include <protobuf/grSim_Commands.pb.h>
 #include <protobuf/grSim_Packet.pb.h>
 #include <Geometry2d/Util.hpp>
 #include <Network.hpp>
 #include <Robot.hpp>
 #include <Utils.hpp>
 #include <stdexcept>
+#include <math.h>
+
+#define PI 3.14159265
```

Comment:
There's a defin in cmath (and probably somewhere in cpp) that does PI correctly

Reply:
M_PI if you include `<cmath>`